### PR TITLE
Add Styles.Attributes enum for common NSAttributedString attributes

### LINKED ISSUE #856: Common Attributed Strings Attrs in Styles
We should have functions that take a `UIColor for common attributes that we have all over the place  so instead of calling:

```swift
attributes: [
  .font: Styles.Fonts.body,
  .foregroundColor: Styles.Colors.Gray.dark.color
]
```

We can call:

```swift
attributes: .hawkBody(color: Styles.Colors.Gray.dark.color)
```
<sub>i imagine that'd be implemented as an extension to `Dictionary`</sub

or: 

```swift
attributes: Styles.Attributes.body(color: Styles.Colors.Gray.dark.color)
```

### PR DESCRIPTION
**Pull request by spsjvc**
Fixes #856.

### CODE REVIEW

**Comment by rnystrom** [comment]
One problem is that I’m about to replace a lot of the fonts with the new TextStyle type from the StyledText framework. The problem with fonts is they store a size, so dynamic content size changes won’t work (size has to change, this font has to change).

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: Classes/Views/Styles.swift [hunk 1]
```diff
@@ -166,3 +166,90 @@ extension String {
...
 
 }
+
+// MARK: Attributes
+
+extension Styles {
+
+    enum Attributes {
+
+        static func body(color: UIColor) -> [NSAttributedStringKey: Any] {
```
Comment (BasThomas) [inline Classes/Views/Styles.swift]: Can we limit the `color` parameter to only accept `Styles.Colors`? 🤔

**Comment by spsjvc** [comment]
@rnystrom Ok, so should I abandon this for now, until the fonts are replaced?

**Comment by rnystrom** [comment]
Ya let's hold off on this until some of the foundational text refactors are wrapped up.

**Pull request closed without merging**